### DISCUSSION TITLE
Profile Page on the github.com

### DISCUSSION THREAD

**Discussion by Tenison** [community/community#13379] [Other Feature Feedback, Questions, & Ideas]
GitHub does not automatically take you to your profile page when you login. It shows the feed of what you have been up to on github.com which is good. But there is no simple icon like a home button to view your profile easily.

**Comment by jfullstackdev** [comment]
yes, that's the default behavior, as for me, I'm just bookmarking it, so it's one click away

**Reply by Tenison** [reply]
ok... thanks

**Comment by airtower-luna** [comment]
There's the "your profile" link in the user menu (top right):
![grafik](https://user-images.githubusercontent.com/5814919/183259953-9f07fb03-0160-47c1-a824-0ee072c3bb61.png)
